src/main/java/taxi/controller/LogInController.java

```diff
@@ -0,0 +1,37 @@
+package taxi.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import taxi.model.Driver;
+import taxi.service.AuthenticationService;
+import taxi.service.AuthenticationServiceImpl;
+
+public class LogInController extends HttpServlet {
+    private AuthenticationService authenticationService = new AuthenticationServiceImpl();
```

Comment:
lets use `injector` to inject dependency

Reply:
fixed